HardwareDefinitions/avnet_sk_rev2_files_for_SDK_installer/avnet_mt3620_sk_rev2.json

```diff
@@ -0,0 +1,90 @@
+{
+  "Metadata": {
+    "Type": "Azure Sphere Hardware Definition",
+    "Version": 1
+  },
+  "Description": {
+    "Name": "Avnet MT3620 Starter Kit (SK) Rev2",
+    "MainCoreHeaderFileTopContent": [
+      "/* Copyright (c) Microsoft Corporation. All rights reserved.",
+      "   Licensed under the MIT License. */",
+      "",
+      "// This JSON hardware definition file specifies peripheral pinouts",
+      "// of the Rev.2 Avnet MT3620 Starter Kit (SK2). The Rev.2 board can",
+      "// be fitted with an Ethernet Click, it also allows reconfiguration",
+      "// of Click2, Pmod and Grove connectors, by moving config resistors.",
+      "// See Hardware User Guide (pages 22-26) for more details.",
+      "// http://avnet.me/mt3620-kit-UG-V2 ",
```

Comment:
Is the link correct?

Reply:
Yes, this is a short URL that we can change on the backend anytime.  We will update the final page when we release the new hardware and corresponding documentation.